circuits/contracts/groth16/RayTracing4/VerifierRayTracing4.sol

```diff
@@ -163,9 +163,7 @@ library Pairing {
 }
 contract VerifierRayTracing4 {
     using Pairing for *;
-
-    event IsVerified(bool indexed isVerifie, address indexed user, uint256[9] indexed input);
-
+    event IsVerified(bool indexed isVerified, address indexed user, uint256[9] indexed input);
```

Comment:
Maybe we should not index this? this is public polygon right? we would get it un hashed, easier to read.

Reply:
"indexed" means it will be hashed?